mmdet/models/dense_heads/lad_head.py

```diff
@@ -0,0 +1,239 @@
+import torch
+from mmcv.runner import force_fp32
+
+from mmdet.core import bbox_overlaps, multi_apply
+from ..builder import HEADS
+from .paa_head import PAAHead, levels_to_images
+
+
+@HEADS.register_module()
+class LADHead(PAAHead):
+    """Label Assignment Head from the paper: `Improving Object Detection by
+    Label Assignment Distillation <https://arxiv.org/pdf/2108.10520.pdf>`_"""
+
+    @force_fp32(apply_to=('cls_scores', 'bbox_preds', 'iou_preds'))
+    def get_la(self,
```

Comment:
```suggestion
    def get_label_assign(self,
```
The full name would be easier to understand.

Reply:
And the return value name is `label_reassign`, should the function be named as `get_label_reassign`?

Reply:
Thanks, I will change the function name to `get_label_assignment` and the return variable name to `label_assignment_results`